### PR TITLE
Fix loading via URL parameter

### LINKED ISSUE #544: `url` query string parameter is broken
### Description

The OSCAL Viewer application supports a `?url` query string parameter to pass a URL to an OSCAL document directly. This was described in #134 and implemented in #143.

### Environment

_No response_

### Steps to Reproduce

1. Go to any of:
  - https://oscal-viewer.msd.easydynamics.com/system-security-plan?url=https://raw.githubusercontent.com/GSA/fedramp-automation/master/dist/content/templates/ssp/json/FedRAMP-SSP-OSCAL-Template.json
  - https://oscal-viewer-dev.msd.easydynamics.com/catalog?url=https://github.com/usnistgov/oscal-content/blob/main/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json

### Expected behavior

The document specified from the `url` parameter is loaded.

### Actual Behavior

The default document URL from the application configuration is loaded.

### Additional Notes

I am not sure exactly when this broke but it was some time in the last few weeks. This is an important functionality to have working. Regression tests must be added in [`EasyDynamics/oscal-editor-deployment`](https://github.com/EasyDynamics/oscal-editor-deployment) to ensure this isn't unexpectedly broken again.

### PR DESCRIPTION
**Pull request by kylelaker**
Closes #544

### CODE REVIEW

**Comment by kylelaker** [comment]
@tuckerzp Can you make sure I didn't break the tests here?

**Review by Bronstrom — COMMENTED** [review]
@kylelaker It looks like the Editor is broken for all models except Catalogs.

**Comment by tuckerzp** [comment]
> Can you make sure I didn't break the tests editor here?

I tried to load `system-security-plan/cff8385f-108e-40a5-8f7a-82f3dc0eaba8` as well as the profiles and it seems things are broken.

File: src/components/OSCALLoader.js [hunk 1]
```diff
@@ -216,10 +217,7 @@ export default function OSCALLoader(props) {
       setIsResolutionComplete(true);
       setHasDefaultUrl(false);
     } else {
-      setOscalUrl(props.oscalObjectType.defaultUrl);
-      loadOscalData(props.oscalObjectType.defaultUrl);
-      setIsLoaded(false);
-      setIsResolutionComplete(false);
+      setOscalUrl(props.oscalUrl || props.oscalObjectType.defaultUrl);
```
Comment (kylelaker) [inline src/components/OSCALLoader.js:220]: I think this bit is what broke things. @tuckerzp I am not going to have time for a little bit today to work on this. I think if this matches what we do on line 62-63 it should go back to working?

Comment (tuckerzp) [reply src/components/OSCALLoader.js:220]: I will look into this.

**Review by Bronstrom — APPROVED** [review]

**Pull request merged**